game-core/src/main/java/games/strategy/triplea/ui/BattleDisplay.java

```diff
@@ -611,10 +638,10 @@ public void actionPerformed(final ActionEvent e) {
                     final CasualtyDetails response = new CasualtyDetails(killed, damaged, false);
                     casualtyDetails.set(response);
                     dicePanel.removeAll();
-                    actionButton.setEnabled(false);
```

Comment:
Do you know why the button was previously disabled here?
Or if you don't, why did you have to change it?

Reply:
It's not strictly necessary for this PR, but this makes things consistent with the other logic.

This code was both disabling the button and setting it to nullAction. In other code, the button was set to nullAction but not disabled. We could do either both or just setting to nullAction, but this case was inconsistent - so the change is to make it consistent.